From 37a646162316d2d8c79d9d92fe099aa2bbd60d36 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Mon, 3 Sep 2018 17:19:46 +0800
Subject: [PATCH] arm64: dts: rockchip: enable 'rockchip,no-deep-low-power' for
 rk3308-evb

This patch can help us to fix pop after wake up via VAD,
and  enable 'rockchip,no-deep-low-power' on all of
rk3308-evb, not only amic boards.

Change-Id: I07f4674dd8c7fbd400b3c9b265fbaec6bfb5829e
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts | 1 -
 arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v11.dts | 1 -
 arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi     | 1 +
 arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi     | 1 +
 4 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
index 5040d9230165..871f1785340f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
@@ -24,7 +24,6 @@
 };
 
 &acodec {
-	rockchip,no-deep-low-power;
 	rockchip,en-always-grps = <0 1 2>;
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v11.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v11.dts
index 0e732ce3001a..c36dc855a256 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v11.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v11.dts
@@ -23,7 +23,6 @@
 };
 
 &acodec {
-	rockchip,no-deep-low-power;
 	rockchip,en-always-grps = <1 2 3>;
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
index 51acfb182bea..c772b8aed263 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
@@ -264,6 +264,7 @@
 &acodec {
 	status = "okay";
 
+	rockchip,no-deep-low-power;
 	rockchip,loopback-grp = <3>;
 	hp-ctl-gpios = <&gpio0 RK_PA1 GPIO_ACTIVE_HIGH>;
 	spk-ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
index 1d931d311a04..8b12bfea0835 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
@@ -283,6 +283,7 @@
 &acodec {
 	status = "okay";
 
+	rockchip,no-deep-low-power;
 	rockchip,loopback-grp = <0>;
 	hp-ctl-gpios = <&gpio0 RK_PA1 GPIO_ACTIVE_HIGH>;
 	spk-ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
-- 
2.35.3

